circle.yml

```diff
@@ -14,11 +14,12 @@ dependencies:
     - curl https://sdk.cloud.google.com | bash
     - gcloud auth activate-service-account $gcloud_account_email --key-file /tmp/gcloud_key.json
     - gcloud docker -a
+  override:
+    - docker build -t=$registry_root/$image_name:b$CIRCLE_BUILD_NUM .
 
 test:
   override:
-    - bundle exec rake
-    - docker build -t=$registry_root/$image_name:b$CIRCLE_BUILD_NUM .
+    - docker run -t=$registry_root/$image_name:b$CIRCLE_BUILD_NUM bundle exec rake
```

Comment:
Do you mind spinning this off and resolving in another PR, then rebasing? (The `-t` option is not what you want for `docker run` btw).